### PR TITLE
[bugfix] disable sendmail-specific periodic tasks

### LINKED ISSUE #7: periodic tasks for sendmail are not disabled in FreeBSD
such as `daily_clean_hoststat_enable`

### PR DESCRIPTION
**Pull request by trombik**
fixes #7

### CODE REVIEW

File: tasks/install-FreeBSD.yml [hunk 1]
```diff
@@ -34,8 +34,15 @@
...
 
-# TODO periodic
-# daily_clean_hoststat_enable="NO"
-# daily_status_mail_rejects_enable="NO"
-# daily_status_include_submit_mailq="NO"
-# daily_submit_queuerun="NO"
+- name: Disable sendmail-specific tasks
+  lineinfile:
+    dest: /etc/periodic.conf
+    regexp: "^{{ item }}="
```
Comment (mitsururike) [inline tasks/install-FreeBSD.yml]: - does regexp: insert the line if not found?
- regexp: 'only the last line found will be replaced.'

File: tests/serverspec/default_spec.rb [hunk 2]
```diff
@@ -41,33 +48,59 @@
...
+  it { should be_owned_by default_user }
+  it { should be_grouped_into default_group }
   it { should be_mode 755 }
 end
 
 describe file("#{ db_dir }/Makefile") do
   it { should exist }
   it { should be_file }
+  it { should be_owned_by default_user }
+  it { should be_grouped_into default_group }
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb]: be_mode 644?

**Review by mitsururike — COMMENTED** [review]
- i have commented in-line.
- rubocop tests/serverspec/default_spec.rb

File: tasks/install-FreeBSD.yml (hunk 1 above)
Comment (trombik) [reply tasks/install-FreeBSD.yml]: for the first question, yes.

the second question made me realize that there is a case where the logic fails (multiple daily_submit_queuerun="NO").

File: tasks/install-FreeBSD.yml [hunk 3]
```diff
@@ -34,13 +34,26 @@
     name: "{{ postfix_service }}"
     enabled: yes
 
+- name: Remove sendmail-specific tasks with YES
+  lineinfile:
+    dest: /etc/periodic.conf
+    regexp: '^{{ item }}="?[Yy]Ee][Ss]"?'
```
Comment (mitsururike) [inline tasks/install-FreeBSD.yml]: s/Ee/[Ee/

**Pull request merged**